### PR TITLE
Star 226 action bar tests

### PR DESCRIPTION
**Pull request by KaanDarcey**
### Purpose 🚀
To create tests for the ActionBar component.

### Notes ✏️
- Created ActionBar.cypress.js
- Created ActionBar.spec.js

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [x] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/STAR-226-ActionBar-tests

### Screenshots 📸
N/A
### References 🔗
N/A

### CODE REVIEW

File: packages/ActionBar/tests/ActionBar.spec.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  it("Should have Filter.rules.CONTAINS set to true if testvalue is empty", () => {
+    expect(testers[Filter.rules.CONTAINS]("any value", "")).toBe(true);
+  });
+
+  it("Should have Filter.rules.IS_BLANK set to true if value is undefined, null, or empty", () => {
+    expect(testers[Filter.rules.IS_BLANK](null)).toBe(true);
+    expect(testers[Filter.rules.IS_BLANK](undefined)).toBe(true);
+    expect(testers[Filter.rules.IS_BLANK]("")).toBe(true);
+    expect(testers[Filter.rules.IS_BLANK]("any value")).toBe(false);
```
Comment (allison-c) [inline packages/ActionBar/tests/ActionBar.spec.js:26]: maybe adding 0 should be false?

File: packages/ActionBar/tests/ActionBar.spec.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    const a = 2;
+    const b = 1;
+    expect(compareNumber(a, b)).toBe(1);
+  });
+
+  it("Should display the date/time differance", () => {
+    const a = "1994-06-05";
+    const b = "1994-09-08";
+    // differance displayed in seconds
+    expect(compareDate(a, b, "YYYY-MM-DD")).toBe(-8208000);
```
Comment (allison-c) [inline packages/ActionBar/tests/ActionBar.spec.js:71]: You could import `compareNumber ` and `compareDate ` from packages/ActionBar/src/hooks/useSort/sort.js

File: packages/ActionBar/tests/Actionbar.cypress.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+describe("ActionBar", () => {
+  beforeEach(() => {
+    cy.visitStorybook("action-bar--showcase");
+  });
+
+  it("Should render", () => {
+    cy.get("span")
```
Comment (allison-c) [inline packages/ActionBar/tests/Actionbar.cypress.js:7]: I think using `data-pka-anchor` is better here, you can use `cy.getByTestId('raw-button')`

File: packages/ActionBar/tests/Actionbar.cypress.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    cy.contains("Add a field to filter by").click();
+    cy.get("select")
+      .eq(0)
+      .select("Goals")
+      .select("Name")
+      .select("Status")
+      .select("Country")
+      .select("Joined by")
+      .select("Shareable")
+      .select("Level");
```
Comment (allison-c) [inline packages/ActionBar/tests/Actionbar.cypress.js:31]: just curious why do you want to select each item once

Comment (KaanDarcey) [reply packages/ActionBar/tests/Actionbar.cypress.js:31]: Yes, was checking to see if all of the items were there.

**Comment by nahumzs** [comment]
is this branch ready to be approved @allison-c ?

**Pull request merged**